internal/pkg/deploy/cloudformation/pipeline.go

```diff
@@ -34,7 +36,28 @@ func (cf CloudFormation) CreatePipeline(in *deploy.CreatePipelineInput) error {
 	if err != nil {
 		return err
 	}
-	return cf.cfnClient.CreateAndWait(s)
+	err = cf.cfnClient.CreateAndWait(s)
```

Comment:
Do we not have unit tests for `CreatePipeline`? Is it possible to add some 😬

Reply:
There were none and I had begun to add some, but it was taking me so long that I decided to open this PR first, then add them. Will do!

Reply:
done.